src/WebUI/ImageDetails/ImageDetails.tsx

```diff
@@ -152,51 +123,47 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
         imageSize && imageDetailsRows.push({ key: Resources.ImageSizeText, value: imageSize });
         labels && labels.length > 0 && imageDetailsRows.push({ key: Resources.LabelsText, value: labels });
 
-        return new ArrayItemProvider<any>(imageDetailsRows);
+        return imageDetailsRows;
     }
 
-    private static _renderValueCell(
-        rowIndex: number,
-        columnIndex: number,
-        tableColumn: ITableColumn<any>,
-        tableItem: any): JSX.Element {
+    private static _renderValueCell = (tableItem: any): JSX.Element => {
         const { key, value } = tableItem;
-        let props: any = {};
-        const contentClassName = "image-o-v-col-content";
         switch (key) {
             case Resources.LabelsText:
             case Resources.TagsText:
-                props = {
-                    columnIndex: columnIndex,
-                    children:
-                    <Tags items={value} />,
-                    tableColumn: tableColumn,
-                    contentClassName: css(contentClassName, "image-labelgroups")
-                };
-
-                return renderSimpleTableCell(props);
+                return (
+                    <div className="text-ellipsis details-card-value-field-size">
+                        <Tags items={value} />
+                    </div>
+                );
 
             default:
-                const itemToRender = <span className="image-details-value-cell">{value}</span>;
-                return renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender, undefined, contentClassName);
+                const itemToRender = defaultColumnRenderer(value);
+                return (
+                    <div className="text-ellipsis details-card-value-field-size">
+                        {itemToRender}
+                    </div>
+                );
         }
     }
 
-    private static _renderKeyCell(
-        rowIndex: number,
-        columnIndex: number,
-        tableColumn: ITableColumn<any>,
-        tableItem: any): JSX.Element {
-        const { key } = tableItem;
-        const contentClassName = "image-o-k-col-content";
-
-        const itemToRender = (
-            <Tooltip text={key} overflowOnly>
-                <span className={css("text-ellipsis")}>{key}</span>
-            </Tooltip>
+    private _getCardContent = (): JSX.Element => {
+        const items = this._getImageDetailsRowsData(this.props.imageDetails);
+        return (
+            <div
+                className="flex-column"
+                style={{ flexWrap: "wrap", width: "100%" }}
+            >
+                {items.map((item, index) => (
+                    <div className="flex-row details-card-row-size bolt-list-cell" key={index}>
```

Comment:
bolt-list-cell --> do we need this css?

Reply:
Removed bolt-list-cell and added relative position to details-card-row-size